src/middlewared/middlewared/plugins/account.py

```diff
@@ -273,6 +277,8 @@ async def query(self, filters, options):
                     'sid': u['User SID'],
                 }})
 
+        group_role_info = {g['id']: g['roles'] for g in await self.middleware.call('group.query')}
```

Comment:
We can user `user_extend_context` and `user_extend` methods to join this information using a more uniform way

Reply:
Ah, good point. I had initially made this configurable.